### ISSUE TITLE
Will not start

### ISSUE THREAD

**Issue by culturedev** [LS31/qrscan#8]
Would be nice, but it throws an error while trying to start:

>java qrscan-2.1.0-jar-with-dependencies.jar
Error: Could not find or load main class qrscan-2.1.0-jar-with-dependencies.jar
Caused by: java.lang.ClassNotFoundException: qrscan-2.1.0-jar-with-dependencies.jar

>java -version
openjdk version "14.0.2" 2020-07-14
OpenJDK Runtime Environment (build 14.0.2+12-46)
OpenJDK 64-Bit Server VM (build 14.0.2+12-46, mixed mode, sharing)

**Comment by LS31** [comment]
Could you please try the following?
`java -jar qrscan-2.1.0-jar-with-dependencies.jar`

**Comment by culturedev** [comment]
Hi, that works - many thanks! (sorry for my ignorance re java apps). Unfortunately, the recognition rate of the app is very low (only recognized 4 out of 24 in my test run). Can this be improved? We would like to use your app for a COVID19 contact tracing application. Could I send you my test sample?

This is what I am seeing in output:
Found:
10:52:34.159 INFO: Now scanning file 7_PDFsam_Scan12102020_092734.pdf.
Okt. 12, 2020 10:52:34 AM org.apache.pdfbox.contentstream.PDFStreamEngine operatorException
SEVERE: Cannot read JBIG2 image: jbig2-imageio is not installed
10:52:34.488 INFO: Found QR code 1000235 in 7_PDFsam_Scan12102020_092734.pdf.

Did not find:
10:52:34.493 INFO: Now scanning file 8_PDFsam_Scan12102020_092734.pdf.
Okt. 12, 2020 10:52:34 AM org.apache.pdfbox.contentstream.PDFStreamEngine operatorException
SEVERE: Cannot read JBIG2 image: jbig2-imageio is not installed
Okt. 12, 2020 10:52:35 AM org.apache.pdfbox.contentstream.PDFStreamEngine operatorException
SEVERE: Cannot read JBIG2 image: jbig2-imageio is not installed
Okt. 12, 2020 10:52:35 AM org.apache.pdfbox.contentstream.PDFStreamEngine operatorException
SEVERE: Cannot read JBIG2 image: jbig2-imageio is not installed
Okt. 12, 2020 10:52:35 AM org.apache.pdfbox.contentstream.PDFStreamEngine operatorException
SEVERE: Cannot read JBIG2 image: jbig2-imageio is not installed
10:52:36.056 WARN: !Unable to find QR code at specified page in 8_PDFsam_Scan12102020_092734.pdf.: com.google.zxing.NotFoundException

**Comment by LS31** [comment]
No problem, I'm glad it now runs.

Unfortunately, I had trouble with recognition as well. I made this app for a specific project with a lot of PDF files. I attained good recognition when I had scanned documents at 300 DPI in grayscale/black-white. Hopefully that helps.

As for "jbig2-imageio is not installed", I will try to look into this if it is a problem with the app - but I currently don't have the time.
